### DISCUSSION TITLE
Does deno serve support express?

### DISCUSSION THREAD

**Discussion by thesiti92** [denoland/deno#26268] [Q&A]
Thinking of porting a cpu-bound node express service at our company to Deno to leverage the multi-threaded deno serve functionality, and i was wondering if there was any prior art on doing this without rewriting the router to std/http. I haven't been able to find anything online. Thanks!

**Comment by bartlomieju** [comment]
I would say no, but I don't know express that well - is there a way to create express app and invoke its handler programatically?

FYI we're considering adding `deno run --parallel` that would handle this case much more easily.
